src/tls_wrap.cc

```diff
@@ -519,12 +561,17 @@ void TLSWrap::ClearOut() {
 
 
 void TLSWrap::ClearIn() {
+  Debug(this, "Trying to write cleartext input");
   // Ignore cycling data if ClientHello wasn't yet parsed
-  if (!hello_parser_.IsEnded())
+  if (!hello_parser_.IsEnded()) {
+    Debug(this, "Bailing out of ClearIn(), hello_parser_ active");
```

Comment:
I think "Not yet ready for " would be more clear than "Bailing out". The debug is mostly useful while reading the source, but Bailing sounds to me like a permanent state, but its just temporary.

Reply:
I replaced all occurrences of `Bailing out of` with `Returning from`, that seems like the most accurate statement